app/controllers/user_reductions_controller.rb

```diff
@@ -22,13 +22,27 @@ def update
     render json: reduction
   end
 
+  def current_user_reductions
```

Comment:
Did you consider just folding this into `index`, maybe with a new `user_id` param so that even project owners will only see their own reductions?

Reply:
i'm resolving this because it's the same plan @camallen suggests below